### PR TITLE
Features/bug 503

### PR DESCRIPTION
**Pull request by ocampd1**

### CODE REVIEW

File: chucks-pick3-mobile/src/app/components/pick3-draw-date-info-section/pick3-draw-date-info-section.ts [hunk 1]
```diff
@@ -110,7 +112,12 @@ export class Pick3DrawDateInfoSection implements OnInit {
             } else {
                 this.getPastWinningDrawingNumber(this.data.getDrawState(), pick3DrawTime.getDateTime(), pick3DrawTime.getType());
             }
-            this.showCountDownToDrawing = true;
+            const path = location.pathname;
```
Comment (pouncilt) [inline chucks-pick3-mobile/src/app/components/pick3-draw-date-info-section/pick3-draw-date-info-section.ts:115]: Combine duplicate code into single method and reuse the method here.

File: chucks-pick3-mobile/src/app/components/pick3-draw-date-info-section/pick3-draw-date-info-section.ts [hunk 2]
```diff
@@ -229,8 +236,12 @@ export class Pick3DrawDateInfoSection implements OnInit {
                     this.setDrawState(this.data, Pick3DrawTimeCardStateEnum.Pick3DrawTimeCardStateEnum.NOT_DRAWN_YET);
             }
         }
-
-        this.showCountDownToDrawing = false;
+        const path = location.pathname;
```
Comment (pouncilt) [inline chucks-pick3-mobile/src/app/components/pick3-draw-date-info-section/pick3-draw-date-info-section.ts:239]: Combine duplicate code into single method and reuse the method here.

**Review by pouncilt — APPROVED** [review]

File: chucks-pick3-mobile/src/app/components/pick3-draw-date-info-section/pick3-draw-date-info-section.ts (hunk 1 above)
Comment (ocampd1) [reply chucks-pick3-mobile/src/app/components/pick3-draw-date-info-section/pick3-draw-date-info-section.ts]: Reduced to single method and reused.

File: chucks-pick3-mobile/src/app/components/pick3-draw-date-info-section/pick3-draw-date-info-section.ts (hunk 2 above)
Comment (ocampd1) [reply chucks-pick3-mobile/src/app/components/pick3-draw-date-info-section/pick3-draw-date-info-section.ts:237]: Reduced to single method and reused.

**Review by pouncilt — APPROVED** [review]

**Pull request merged**